frontend/public/components/masthead.tsx

```diff
@@ -197,8 +177,5 @@ export type UserMenuProps = {
 
 export type OSUserMenuProps = {
   actions: Actions,
-};
-
-export type OSUserMenuState = {
-  username: string,
+  user?: any,
```

Comment:
nit: this should probably be 

```suggestion
  user?: K8sResourceKind;
```